packages/perseus/package.json

```diff
@@ -65,6 +65,7 @@
         "@khanacademy/wonder-blocks-form": "catalog:",
         "@khanacademy/wonder-blocks-icon-button": "catalog:",
         "@khanacademy/wonder-blocks-icon": "catalog:",
+        "@khanacademy/wonder-blocks-labeled-field": "catalog:",
```

Comment:
We should list this dependency in the `peerDependencies` of this package also. Some commentary that might help: https://khanacademy.atlassian.net/wiki/spaces/LC/pages/2473951425/Should+Perseus+dependencies+be+peer+dependencies